doc/nrf/ug_multi_image.rst

```diff
@@ -217,29 +250,31 @@ You can use the CMake environment variables `VERBOSE`_ and `CMAKE_BUILD_PARALLEL
 
 When using |SES|, you must set these environment variables before starting SES, and they will apply only to the build of the child images.
 On the command line, you must set them before invoking ``west``, and they will apply to both the parent image and the child images.
-For example, to build with verbose output and one parallel job, use the following commands (where *board_name* is the name of the board for which you are building):
+For example, to build with verbose output and one parallel job, use the following commands (where ``<board_name>`` is the name of the board for which you are building):
 
 * Linux/macOS:
 
      .. parsed-literal::
         :class: highlight
 
-        $ VERBOSE=True CMAKE_BUILD_PARALLEL_LEVEL=1 west build -b *board_name*
+        $ VERBOSE=True CMAKE_BUILD_PARALLEL_LEVEL=1 west build -b <board_name>
```

Comment:
Why not do this the proper CMake way ?

```suggestion
        west build -b <board_name> -- -DCMAKE_VERBOSE_MAKEFILE=1 -DCMAKE_BUILD_PARALLEL_LEVEL=1
```

Then you can remove the distinction between Linux/MacOS/Windows, as they all work the same way.

Reply:
Addressed in [this commit](https://github.com/nrfconnect/sdk-nrf/pull/3607/commits/8f78f39ea2d00a43e1149293706416747f825acd) for #3607.